nat/service_ics.go

```diff
@@ -0,0 +1,147 @@
+// +build windows
+
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package nat
+
+import (
+	"net"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type serviceICS struct {
+	ifaces             map[string]struct{} // list in internal interfaces with enabled internet connection sharing
+	remoteAccessStatus string
+}
+
+func (nat *serviceICS) Enable() error {
+	status, err := powerShell("Get-Service RemoteAccess | foreach { $_.StartType }")
```

Comment:
What this 'StartType' means?

Reply:
It's a service start type:
![image](https://user-images.githubusercontent.com/8612618/53626617-66a99f80-3c30-11e9-9020-e1c495840ca8.png)

Reply:
nice table